ratelimit/options.go

```diff
@@ -0,0 +1,16 @@
+// See LICENSE for licensing terms.
```

Comment:
may as well remove all of this and let people view the LICENSE file ... when i get time will clean the rest of the codebase